helm/aws-load-balancer-controller/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-load-balancer-controller
 description: AWS Load Balancer Controller Helm chart for Kubernetes
-version: 1.11.0
+version: 1.11.1
```

Comment:
You dont need to update the version here. We will update it when we cut a release PR.

Reply:
Thx! Fixed.